src/classifier/Alphabet.h

```diff
@@ -83,9 +83,9 @@ class Alphabet : public std::tr1::unordered_map <string, int> {
     }
   }
 
- private:
+private:
```

Comment:
These keywords (private, public, etc.) should actually be 1-space indented. It's a little strange, but that's how it is in https://google.github.io/styleguide/cppguide.html#Classes.